src/index.js

```diff
@@ -25,24 +25,30 @@ class IIIFLambda {
     this.respond = callback;
     this.sourceBucket = sourceBucket;
     this.handled = false;
+    // console.log("IIIFLambda.constructor called with sourceBucket", sourceBucket)
   }
 
   directResponse (result) {
     var base64 = /^image\//.test(result.contentType);
     var content = base64 ? result.body.toString('base64') : result.body;
     var response = {
       statusCode: 200,
-      headers: { 
+      headers: {
         'Content-Type': result.contentType,
         'Access-Control-Allow-Origin': '*'
        },
       isBase64Encoded: base64,
       body: content
     };
+    console.log("base64.length = ", content.length, "original.length = ", result.body.length)
```

Comment:
what i am wondering if we don't want to trap the size error here and call handle error ourselves ?